app/views/govuk_publishing_components/components/_document_list.html.erb

```diff
@@ -14,16 +14,22 @@
             )
           %>
         </h3>
-        <ul>
-          <li class="gem-c-document-list__attribute">
-            <time datetime="<%= item[:metadata][:public_updated_at].iso8601 %>">
-              <%= l(item[:metadata][:public_updated_at], format: '%e %B %Y') %>
-            </time>
-          </li>
-          <li class="gem-c-document-list__attribute gem-c-document-list__attribute--document-type">
-            <%= item[:metadata][:document_type] %>
-          </li>
-        </ul>
+        <p class="gem-c-document-list__item-description" ><%= item[:link][:description] %></p>
```

Comment:
Since the description is optional we should check for it before writing paragraph tags.